unstable/wlr-export-dmabuf-unstable-v1.xml

```diff
@@ -139,11 +142,22 @@
            summary="index of the the plane the data in the object applies to"/>
     </event>
 
+    <event name="sync_fence" since="2">
```

Comment:
We can't add an event in the middle of the protocol, because the event opcode is its index. We generally add news events at the end of the interface, after `<!-- Version 2 additions -->`.